src/mixins/filing-template-mixin.ts

```diff
@@ -14,90 +14,90 @@ import { ActionTypes, CoopTypes, CorrectionErrorTypes, EffectOfOrders, FilingTyp
-  @Action setEntityType!: ActionBindingIF
-  @Action setOfficeAddresses!: ActionBindingIF
-  @Action setNaics!: ActionBindingIF
-  @Action setNameTranslations!: ActionBindingIF
-  @Action setNameRequest!: ActionBindingIF
-  @Action setPeopleAndRoles!: ActionBindingIF
-  @Action setCertifyState!: ActionBindingIF
-  @Action setShareClasses!: ActionBindingIF
-  @Action setEffectiveDateTimeString!: ActionBindingIF
-  @Action setIsFutureEffective!: ActionBindingIF
-  @Action setFolioNumber!: ActionBindingIF
-  @Action setTransactionalFolioNumber!: ActionBindingIF
-  @Action setStaffPayment!: ActionBindingIF
-  @Action setDetailComment!: ActionBindingIF
-  @Action setEntitySnapshot!: ActionBindingIF
-  @Action setDocumentOptionalEmail!: ActionBindingIF
-  @Action setProvisionsRemoved!: ActionBindingIF
-  @Action setNewResolutionDates!: ActionBindingIF
-  @Action setFileNumber!: ActionBindingIF
-  @Action setHasPlanOfArrangement!: ActionBindingIF
-  @Action setSpecialResolution!: ActionBindingIF
-  @Action setCorrectionStartDate!: ActionBindingIF
-
-  @Mutation mutateRestorationApprovalType!: ActionBindingIF
-  @Mutation mutateRestorationCourtOrder!: ActionBindingIF
-  @Mutation mutateRestorationExpiry!: ActionBindingIF
-  @Mutation mutateRestorationType!: ActionBindingIF
+  @Action(useStore) setBusinessContact!: ActionBindingIF
+  @Action(useStore) setBusinessInformation!: ActionBindingIF
+  @Action(useStore) setCorrectionInformation!: ActionBindingIF
+  @Action(useStore) setEntityType!: ActionBindingIF
+  @Action(useStore) setOfficeAddresses!: ActionBindingIF
+  @Action(useStore) setNaics!: ActionBindingIF
+  @Action(useStore) setNameTranslations!: ActionBindingIF
+  @Action(useStore) setNameRequest!: ActionBindingIF
+  @Action(useStore) setPeopleAndRoles!: ActionBindingIF
+  @Action(useStore) setCertifyState!: ActionBindingIF
+  @Action(useStore) setShareClasses!: ActionBindingIF
+  @Action(useStore) setEffectiveDateTimeString!: ActionBindingIF
+  @Action(useStore) setIsFutureEffective!: ActionBindingIF
+  @Action(useStore) setFolioNumber!: ActionBindingIF
+  @Action(useStore) setTransactionalFolioNumber!: ActionBindingIF
+  @Action(useStore) setStaffPayment!: ActionBindingIF
+  @Action(useStore) setDetailComment!: ActionBindingIF
+  @Action(useStore) setEntitySnapshot!: ActionBindingIF
+  @Action(useStore) setDocumentOptionalEmail!: ActionBindingIF
+  @Action(useStore) setProvisionsRemoved!: ActionBindingIF
+  @Action(useStore) setNewResolutionDates!: ActionBindingIF
+  @Action(useStore) setFileNumber!: ActionBindingIF
+  @Action(useStore) setHasPlanOfArrangement!: ActionBindingIF
+  @Action(useStore) setSpecialResolution!: ActionBindingIF
+  @Action(useStore) setCorrectionStartDate!: ActionBindingIF
+  @Action(useStore) setRestorationApprovalType!: ActionBindingIF
+  @Action(useStore) setRestorationCourtOrder!: ActionBindingIF
+  @Action(useStore) setRestorationExpiry!: ActionBindingIF
+  @Action(useStore) setRestorationType!: ActionBindingIF
```

Comment:
1. Any idea if there's a way to import the entire getters/actions instead?

2. I've seen where components that use this mixin inherit these imports, but I don't really like that. I think it's better for components to be explicit and precise about what getters and actions they use.

3. Maybe point 2 above means we shouldn't do point 1... thoughts?

Reply:
1. I bet there probably is a way to import all of the getters / actions. 

2. You get typescript errors - it asks you to use declare to override 

3. You could change it all you want after this PR. I just wanted to maintain similar structure, that way people don't get too side tracked.

Reply:
OK, don't worry about it. Thanks for looking into it.